doc/development.md

```diff
@@ -76,13 +76,14 @@ dart pub global run mono_repo pub get
 ## Local Development with Google AppEngine
 
 To run the default application (web frontend) locally, do the following steps:
-```
+```bash
 cd app
 dart pub get
 export GCLOUD_PROJECT=<gcloud-project>
 export GCLOUD_KEY=<path-to-service-account-key.json>
 dart bin/server.dart default
 ```
+Any project with prefix `dartlang-pub-dev` will be considered as a `dev` env.
```

Comment:
Let's remove this for now, we don't want to establish such pattern - it could lead to weird name collisions and name squatting.